variables.tf

```diff
@@ -216,6 +223,19 @@ variable "kubernetes_version" {
   default     = null
 }
 
+variable "load_balancer_profile" {
+  type        = bool
+  description = "(Optional) A load_balancer_profile block. This can only be specified when load_balancer_sku is set to standard."
+  default     = null
+  nullable    = true
```

Comment:
nit: if the default is `null` you can omit the `nullable = true`.
asking @lonegunmanb for confirmation on this style issue